test/puppetlabs/ring_middleware/core_test.clj

```diff
@@ -626,7 +628,7 @@
               response (stack (basic-request))
               json-body (json/parse-string (response :body))]
           (is (= 500 (response :status)))
-          (is (logged? #".*Internal Server Error.*" :warn))
+          (is (logged? #".*Internal Server Error.*" :error))
```

Comment:
I'm not sure how this was passing before since the message is being logged at an `:error` level.  Maybe the version of the `logged?` function which was previously being used was mishandling the log level?  Seems like we had a defect around that at one point but I can't remember for sure now.